checked_message.go

```diff
@@ -83,10 +91,12 @@ func (m *CheckedMessage) Write(fields ...Field) {
 	case FatalLevel:
 		m.logger.Fatal(m.msg, fields...)
 	default:
-		m.logger.Log(m.lvl, m.msg, fields...)
+		m.logger.Log(t, m.lvl, m.msg, fields...)
```

Comment:
Can we leave a TODO to use `logger.Log` and propagate time in the above cases too? Probably best to address that when we make it easier to write new loggers (i.e., when we're 100% sure that all `logger.Panic` methods just call `logger.Log` and the `panic`).